README.md

```diff
@@ -0,0 +1,66 @@
+<img src="/doc-base/neo.png" align="right" height="70"/>
+
+# NEO Specification
+
+## document format
+The Community Specification Paper base document started as pure LaTeX (as a [Yellow Paper](https://github.com/NeoResearch/yellowpaper])), as usual in academic publications, however the syntax is not considered as friendly as Markdown. So, why not merge both? :)
+In order to facilitate the contributions to the document, we decided to adopt pandoc format, which allows hybrid LaTeX + Markdown syntax.
+
+### base structure
+The structure of the document consists of five files:
+- specification_paper.md : main document in Mardown + LaTeX (this file is automatically generated by `merge_sections.sh`)
+- [metadata.yaml](doc-base/metadata.yaml) : document metadata (authors, abstract, ...)
+- [style_specification_paper.pandoc](doc-base/style_specification_paper.pandoc) : document style (LaTeX headings)
+- [eisvogel.latex](doc-base/eisvogel.latex) : base LaTeX template (Eisvogel template with small changes)
+- [references.bib](doc-base/references.bib) : bibliography in BibTeX format
+
+Specification Paper sections are separated on `sections/` folder (these are used as source to generate `specification_paper.md` file).
+
+Current draft proposal includes 11 sections + references (this may be changed by community, as anything else!):
+- [01_Introduction.md](sections/01_Introduction.md) : expected release date -> Q2 2019
+- [02_Network.md](sections/02_Network.md) : expected release date -> Q2+ 2019
+- [03_Numbers.md](sections/03_Numbers.md) : expected release date -> Q2 2019
+- [04_Cryptography.md](sections/04_Cryptography.md) : expected release date -> Q2 2019
+- [05_Assets.md](sections/05_Assets.md) : expected release date -> Q2+ 2019
+- [06_Transactions.md](sections/06_Transactions.md) : expected release date -> Q2+ 2019
+- [07_NeoVM_NeoContract.md](sections/07_NeoVM_NeoContract.md) : expected release date -> Q2 2019
+- [08_dBFT.md](sections/08_dBFT.md) : first released 14 March, 2019. **PDF:** [Delegated Byzantine Fault Tolerance: Technical details, challenges and perspectives](releases/08_dBFT.pdf)
+- [09_SmartEconomy.md](sections/09_SmartEconomy.md) : expected release date -> Q2+ 2019
```

Comment:
Can we remove all the prefix number? If we need to insert a new section, we have to change each number.

Reply:
Done.